dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ErrorType.java

```diff
@@ -22,25 +22,35 @@
   CLASS_NOT_FOUND,
 
   /**
-   * The referenced class or interface found in the class path is not compatible with the source.
+   * The referenced class or interface found in the class path is not binary-compatible with the
+   * source class.
+   *
+   * <p>An example case of breaking binary-compatibility is when a superclass changes a method to
+   * {@code final} and subclass is still overriding the method. Another example is when there is a
+   * method call to an interface and the interface is changed to a class.
```

Comment:
This doesn't sound quite right.

Reply:
I'm trying to explain "If C is not an interface, interface method resolution throws an IncompatibleClassChangeError." here
https://docs.oracle.com/javase/specs/jvms/se8/html/jvms-5.html#jvms-5.4.3.4